client/src/components/EmployeeTable/index.jsx

```diff
@@ -0,0 +1,223 @@
+            inputProps={{ 'aria-label': 'Select all employees' }}
+          />
+        </TableCell>
+        {headCells.map((headCell) => (
+          <TableCell
+            key={headCell.id}
+            align={headCell.numeric ? 'right' : 'left'}
+            padding={headCell.disablePadding ? 'none' : 'default'}
+          />
+        ))}
+      </TableRow>
+    </TableHead>
+  );
+};
+
+const EnhancedTableToolbar = () => {
+  const { recipients } = useSelector((state) => state.employeeTableReducer);
+
+  const numSelected = recipients.length;
+  return (
+    <Toolbar>
+      {numSelected > 0 ? (
+        <Box
+          width={1}
+          display='flex'
+          justifyContent='center'
+          style={{ backgroundColor: '#f443361f' }}
+        >
+          <Typography align='center' color='inherit' variant='subtitle1'>
+            {`${numSelected} selected`}
+          </Typography>
+        </Box>
+      ) : (
+        <Typography variant='h5'>Employees</Typography>
+      )}
+    </Toolbar>
+  );
+};
+
+const EmployeesTable = () => {
+  const { filteredEmployeeData, recipients, page, rowsPerPage } = useSelector(
+    (state) => state.employeeTableReducer,
+  );
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const getEmployees = async () => {
+      const { data } = await axios.get(`/employees`);
+      dispatch({ type: 'SET_EMPLOYEE_DATA', payload: data });
+      dispatch({ type: 'SET_FILTERED_EMPLOYEE_DATA', payload: data });
+      dispatch({ type: 'SET_INTITAL_FILTER_OPTIONS' });
+    };
+    getEmployees();
+  }, [dispatch]);
+
+  const handleRowClick = (event, id) => {
+    const payload = { id };
+    dispatch({ type: 'TOGGLE_RECIPIENT', payload });
+  };
+
+  const handleChangePage = (event, newPage) => {
+    const payload = { page: newPage };
+    dispatch({ type: 'CHANGE_PAGE', payload });
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    const payload = { rowsPerPage: event.target.value };
+    dispatch({ type: 'CHANGE_ROWS_PER_PAGE', payload });
+  };
+
+  const isSelected = (id) => recipients.indexOf(id) !== -1;
+
+  const emptyRows = filteredEmployeeData
+    ? rowsPerPage -
+      Math.min(
+        rowsPerPage,
+        filteredEmployeeData.length - (page - 1) * rowsPerPage,
+      )
+    : 0;
+
+  return (
+    <Box pt={2}>
+      <RecipientOptions />
+      {filteredEmployeeData && (
+        <>
+          <EnhancedTableToolbar />
+          <Box>
+            <Table
+              aria-labelledby='tableTitle'
+              size='small'
+              aria-label='enhanced table'
+            >
+              <EnhancedTableHead />
+
+              <TableBody>
+                {filteredEmployeeData
+                  .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+                  .map((row) => {
+                    const isItemSelected = isSelected(row.id);
+                    const labelId = row.id;
+
+                    return (
```

Comment:
If I set the results to return as 15, I am not able to scroll down the modal and see the extra employees.